### DISCUSSION TITLE
Question about Paper Submission

### DISCUSSION THREAD

**Discussion by Fantasydog05** [DLR-VSDC/IEEE-MVC-2023#36] [Q&A] [answered]
Hello, 

I'd like to confirm if paper submission is required. Is it only for competitors who are selected or for everyone who participates in the competition?

Thank you for your reply.

**Comment by JBRDLR** [comment, marked as answer]
We encourage to submit a paper to the VPPC 2023. That would be great acknowledge for all the work we did here.
@rpintodecastro

**Comment by rpintodecastro** [comment]
As mentioned by @JBRDLR, paper submission is encouraged but not required.
